.github/workflows/push-on-develop.yml

```
@@ -18,6 +18,7 @@ jobs:
         include:
           - tag: preprod-fr
             NUXT_PUBLIC_WEREWOLVES_ASSISTANT_API_BASE_URL: "https://preprod.api.werewolves-assistant.com"
+            NUXT_API_KEY: ${{ secrets.PREPRODUCTION_WEREWOLVES_ASSISTANT_API_KEY }}
```

actionlint

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)